packages/backend/src/managers/SnippetManager.spec.ts

```diff
@@ -0,0 +1,84 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { beforeEach, expect, test, vi } from 'vitest';
+import { SnippetManager } from './SnippetManager';
+import { getLanguageList, convert } from 'postman-code-generators';
+import type { Webview } from '@podman-desktop/api';
+import { Request } from 'postman-collection';
+import { MSG_SUPPORTED_LANGUAGES_UPDATE } from '@shared/Messages';
+vi.mock('postman-code-generators', () => {
```

Comment:
it's nice to mock here for the manager tests, but I think we're missing a test to really test the postman-code-generators

For example, if there is an automated PR bumping the version of postman-code-generators, there won't be any test that will check the behavior of the library

I think then we should use nock to mock a request and see how postman-code-generators behave

Reply:
I adapted the tests

Reply:
thanks, we'll able to trust tests in case of version update 🎉